src/transformState.js

```diff
@@ -6,6 +6,24 @@
  */
 function transformState(state, actions) {
   // write code here
+  actions.forEach(action => {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+      case 'removeProperties':
+        action.keysToRemove.forEach((key) => {
+          delete state[key];
+        });
+        break;
+      case 'clear':
+        Object.keys(state).forEach((key) => delete state[key]);
+        break;
+
+      default:
+        break;
```

Comment:
![image](https://github.com/mate-academy/js_stateful-object/assets/54860641/d7477526-b1c3-4669-b3b3-d6b8d939306e)
For this task, you can return some message that something went wrong.